packages/react-scripts/config/jest/graphqlTransform.js

```diff
@@ -8,6 +8,10 @@
 // @remove-on-eject-end
 'use strict';
 
```

Comment:
Please add `Copyright (c) 2016 Remind` to the license header of this file, per
https://unpkg.com/jest-transform-graphql@2.1.0/LICENSE

Reply:
Does https://github.com/facebook/create-react-app/pull/3909/commits/39d39bcc57650c3db27394785e0c45e2c8be877d look correct? Not  100% sure how derivative works should be marked.